cpp/test/Ice/scope/AllTests.cpp

```diff
@@ -105,24 +105,24 @@ allTests(Test::TestHelper* helper)
         Test::CPtr c1 = make_shared<Test::C>(s1);
         {
             auto result = i->opCAsync(c1).get();
-            test(Ice::targetEqualTo(std::get<0>(result), c1));
-            test(Ice::targetEqualTo(std::get<1>(result), c1));
+            test(std::get<0>(result)->ice_tuple() == c1->ice_tuple());
```

Comment:
If you need/want these comparisons, you can compare the tuples. Or you can write your own relational operators that compare the tuples (+ ice_id() and more, it's up to you).